CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## Unreleased
+### Changed
```

Comment:
if this is a breaking change there should be a heading here to denote that

Reply:
Adding "breaking change" as a header in 65327df.